examples/export_figure.py

```diff
@@ -0,0 +1,122 @@
+"""
+Export Figure
+=============
+
+Display one shapes layer ontop of one image layer using the ``add_shapes`` and
+``add_image`` APIs. When the window is closed it will print the coordinates of
+your shapes.
+
```

Comment:
I think this example should be reduced to just a few lines, there's way too much stuff before the actual message (`scaled_export_figure = viewer.export_figure(scale=5)`) and most of it is unrelated.

Reply:
It is mostly comparable to the `to_screenshot` example, but I can make it smaller.

Reply:
I agree with @melonora, let's not get too bogged down in how big or small an example is — we can always tweak it later. As long as it shows the functionality, it should be ok.

Reply:
Yeah, not blocking.